cmd/crictl/container.go

```diff
@@ -508,19 +508,82 @@ var containerStatusCommand = &cli.Command{
 			Name:  "template",
 			Usage: "The template string is only used when output is go-template; The Template format is golang template",
 		},
+		&cli.StringFlag{
+			Name:  "name",
+			Value: "",
```

Comment:
Isn't an empty string the default value? It would be for a string variable.

Reply:
Yeah that's a copy paste leftover, fixed as suggested.